configuration/concepts/diagnoses.csv

```diff
@@ -0,0 +1,582 @@

```

Comment:
Sorry, belated comment here... I think @brandones and @mseaton  were going back and forth on this, and I agree with @mseaton  that having a giant cell of semicolon delimited concept names to maintain set membership does seem like a real pain to maintain (vs a separate file).  However, I think I'd also be okay with starting like this and seeing if our fears are warrantied.